dog-blog-page3.html

```diff
@@ -0,0 +1,61 @@
+<!--Adopt-a-Dog's blog Page 3 -->
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Adopt-a-Dog</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <!--this is for the future style sheet-->
+    <link rel="stylesheet" type="text/css" href="styles.css">
+  </head>
+  <body>
+    <header>
+      <h1>ADOPTADOG</h1>
+        <img src="adoptadog-logo.svg" width="500" alt="white dog silhoutte">
```

Comment:
nice alt tag, for now the styling in the HTML is fine, but make sure to remove it when you have a css file.